### PR TITLE
Pfappserver Updates

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Continue work on the pfappserver

# Impacts
Increased PR count :100: 

# Delete branch after merge
YES

### CODE REVIEW

File: html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationMaintenanceTasks.js [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+          attrs: pfConfigurationAttributesFromMeta(meta, 'delay.unit'),
+          validators: pfConfigurationValidatorsFromMeta(meta, 'delay.unit', 'Unit')
+        }
+      ]
+    }
+  },
+  delete_window: ({ options: { meta = {} } } = {}) => {
+    return {
+      label: i18n.t('Delete window'),
+      text: i18n.t(`How long can an unregistered node be inactive on the network before being deleted.\nThis shouldn't be used if you are using port-security`),
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationMaintenanceTasks.js]: End the sentence with a period.

File: html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationMaintenanceTasks.js [hunk 2]
```diff
@@ -0,0 +1,667 @@
...
+          fields: [
+            pfConfigurationMaintenanceTaskFields.id(context),
+            pfConfigurationMaintenanceTaskFields.description(context),
+            pfConfigurationMaintenanceTaskFields.status(context),
+            pfConfigurationMaintenanceTaskFields.interval(context)
+          ]
+        }
+      ]
+    default:
+      return [
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationMaintenanceTasks.js:660]: We should probably raise an exception if the task id is not known.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/FilterEngineTabs.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<template>
+  <b-card no-body>
+    <b-card-header>
+      <h4 v-t="'Filter Engines'"></h4>
+      <p v-t="'This is an advanced section. Proceed with caution when editing.'"></p>
+    </b-card-header>
+    <b-tabs ref="tabs" v-model="tabIndex">
+      <b-tab v-for="filter in filters" :key="filter" :title="$t(filter).toUpperCase()" class="h-100 mb-3" @click="changeTab(filter)">
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/FilterEngineTabs.vue]: We should rely on the unmodified translated string. Don't uppercase it.

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/RolesList.vue [hunk 4]
```diff
@@ -14,9 +14,9 @@
       </template>
       <template slot="buttons" slot-scope="item">
         <span class="float-right text-nowrap">
-          <b-button size="sm" variant="outline-primary" class="mr-1" :to="{ name: 'TODO' }">{{ $t('Traffic Shaping') }}</b-button>
           <b-button size="sm" variant="outline-primary" class="mr-1" @click.stop.prevent="clone(item)">{{ $t('Clone') }}</b-button>
           <pf-button-delete  v-if="!item.not_deletable" size="sm" variant="outline-danger" :disabled="isLoading" :confirm="$t('Delete Role?')" @on-delete="remove(item)" reverse/>
+          <b-button size="sm" variant="outline-primary" class="mr-1" :to="{ name: 'TODO' }">{{ $t('Traffic Shaping') }}</b-button>
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/RolesList.vue]: Traffic shaping should be editable from the `RoleView.vue` component in a designated tab, but not in a separate modal.

**Review by cgx — CHANGES_REQUESTED** [review]

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/RolesList.vue (hunk 4 above)
Comment (satkunas) [reply html/pfappserver/root/static.alt/src/views/Configuration/_components/RolesList.vue]: Traffic Shaping will redirect the user to the traffic shaping section.

**Review by cgx — APPROVED** [review]

**Pull request merged**